tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -3187,6 +3187,122 @@ public function testContentTypeSwitch()
         $this->mapper->save($testSiteData, 'internal-link', 'sulu_io', 'en', 1, true, $uuid);
     }
 
+    /**
+     * It should delete a node which has children with history.
+     * It should not throw an exception
```

Comment:
@chirimoya @wachterjohannes @danrot I would vote +1 to make these comments mandatory for all future tests. They should read as specifications for the functionality. `It should do this`, `It should do that`.